lib/diagnostic.rb

```diff
@@ -106,10 +110,15 @@
 # Use shorthand to make this one line long.
 # Write your response as a Ruby string.
 
-Diagnostic.person_hash = 'your answer here'
+Diagnostic.person_hash = 'class Person
+  def initialize(first_name, last_name)
+  @first_name = first_name
+  @last_name = last_name
+  end
+  Person.first_name('leslie').last_name('knope')'
 
```

Comment:
`person = {first_name: 'Ben', age: 28}`

we were looking for this, does that maks sense?